src/method.c

```diff
@@ -735,6 +741,9 @@ JL_DLLEXPORT jl_code_info_t *jl_code_for_staged(jl_method_instance_t *linfo, siz
                 }
                 jl_error("The function body AST defined by this @generated function is not pure. This likely means it contains a closure, a comprehension or a generator.");
             }
+            // TODO: This should ideally be in the lambda expression,
```

Comment:
This is impossible (as it is defined by value not syntax), so it should not be a TODO

Reply:
Potentially both `...` and `Vararg{` could be recognized syntactically without being too breaking, but I wouldn't want to do it without a bigger motivation than this. Or in 2.0 of course.

Reply:
Breaking design changes should not have TODO comments though. The syntactic thing is possible, but breaks referential transparency and seems to gain nothing but a slightly more convoluted lowering, so I don't think that seems worthwhile